api/devices/schema.py

```diff
@@ -62,7 +62,7 @@ class Query(graphene.ObjectType):
 
     def resolve_all_devices(self, info):
         query = Devices.get_query(info)
-        return query.all()
+        return query.order_by(DevicesModel.name).all()
```

Comment:
is there a reason this is not sorted using the lower function?

Reply:
@omobosteven , There is no reason. I just missed modifying that part. Thanks for pointing that out. It is all done now